### PR TITLE
Translation for files in token folder in containers.

### PR DESCRIPTION
**Pull request by MilanPospisil**
Translation for files in token folder in containers.

### CODE REVIEW

File: src/containers/token/token-insights.tsx [hunk 1]
```diff
@@ -58,36 +58,39 @@ class TokenPage extends React.Component<RouteComponentProps, IState> {
...
-              <a
-                href='https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/'
-                target='_blank'
-              >
-                here
-              </a>
-              . Use the following parameters to configure the client.
-            </p>
+            <Trans>
+              <h2>Connect the ansible-galaxy client</h2>
```
Comment (himdel) [inline src/containers/token/token-insights.tsx:74]: the heading should not be part of the text below, separate trans

File: src/containers/token/token-insights.tsx [hunk 2]
```diff
@@ -113,33 +116,37 @@ class TokenPage extends React.Component<RouteComponentProps, IState> {
               </ClipboardCopy>
             </div>
             <h2>{t`Manage tokens`}</h2>
-            To revoke a token or see all of your tokens, visit the{' '}
+            <Trans>
+              To revoke a token or see all of your tokens, visit the{' '}
+            </Trans>
```
Comment (himdel) [inline src/containers/token/token-insights.tsx:121]: trans ending in the middle of sentence .. the link and "page" need to be in the same one

File: src/containers/token/token-insights.tsx [hunk 3]
```diff
@@ -113,33 +116,37 @@ class TokenPage extends React.Component<RouteComponentProps, IState> {
...
+              download content from Automation Hub.`}
             </p>
             <ClipboardCopy isReadOnly>{getRepoUrl('')}</ClipboardCopy>
             <p>
-              {t`Note: this URL contains all collections in Hub. To connect to your`}
-              organization's sync repository use the URL found on{' '}
+              <Trans>
+                Note: this URL contains all collections in Hub. To connect to
+                your organization's sync repository use the URL found on{' '}
+              </Trans>
```
Comment (himdel) [inline src/containers/token/token-insights.tsx:141]: and here, the link should be part of the text

**Comment by MilanPospisil** [comment]
Closing in favor #790

**Pull request closed without merging**